src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -136,5 +137,30 @@ public static Set<Integer> parseShiftDays(String shiftDayString) {
                 .boxed().collect(Collectors.toSet());
     }
 
+    /**
+     * Parses {@code String specialisations} into a {@code Set<Specialisation>}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code specialisations} is invalid.
+     */
+    public static Set<seedu.address.model.person.Specialisation> parseSpecialisations(String specialisations)
+            throws ParseException {
+        requireNonNull(specialisations);
+        String trimmedSpecialisations = specialisations.trim();
+        if (trimmedSpecialisations.isEmpty()) {
+            return new HashSet<>();
+        }
+        String[] specialisationNames = trimmedSpecialisations.split(",");
```

Comment:
One concern I can think of with this is what if the user intends to add a specialisation that contains `,`? 
What do you think?

Reply:
Hmm, comma was just so that it is standardised with Affiliations, but we can def discuss using other separators like ;
Otherwise, I can add the constraint in the usage description.